lib/commands/context/exports.js

```diff
@@ -198,6 +198,18 @@ export async function setWindow(handle) {
   await chromedriver.jwproxy.command('/window', 'POST', {[paramName]: handle});
 }
 
+/**
+ * Open a new tab/window in web view context (chrome)
+ * @param {string} type
+ * @returns {Promise<import('../types').CreateNewWindow>}
+ */
+export async function createNewWindow(type) {
```

Comment:
isn't it automatically gets proxied to chromedriver? Perhaps we should just tune the corresponding NO_PROXY regexp to not include the correspond endpoint

Reply:
Yea, i noticed that after creating this pr -> https://github.com/appium/appium-uiautomator2-driver/pull/876